prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-actions/adjudicator-actions.component.html

```diff
@@ -106,6 +106,12 @@
       <mat-icon>edit_calendar</mat-icon>
       <span>Change Date of Birth</span>
     </button>
+    <button mat-menu-item
+            *ngIf="(Role.SUPER_ADMIN | inRole )&& canChangeToA()"
```

Comment:
Should this be `Role.MANAGE_ENROLLEE` ... have we checked with Business?

Reply:
Not check with Business yet, but i think you are right. MANAGE_ENROLLEE is better.